ApiBundle/Transformer/AreaTransformer.php

```diff
@@ -94,35 +100,42 @@ public function transform($area, NodeInterface $node = null, $parentAreaId = nul
                 'id' => $area->getAreaId()
             )
         );
+
         $facade->addLink('_self_form', $this->generateRoute('open_orchestra_backoffice_area_form', array(
             'nodeId' => $node->getId(),
             'areaId' => $area->getAreaId(),
         )));
-        $facade->addLink('_self_block', $this->generateRoute('open_orchestra_api_area_update_block', array(
-            'nodeId' => $node->getId(),
-            'areaId' => $area->getAreaId()
-        )));
+
+        if ($this->authorizationChecker->isGranted(TreeNodesPanelStrategy::ROLE_ACCESS_UPDATE_NODE)) {
```

Comment:
Is this done for template editing ? If yes, it means that editing template require the right to edit nodes, which may be a  problem to discuss.